### ISSUE TITLE
Spawn ball hypotheses in Heatmap on kick-in (and corner kick)

### ISSUE THREAD

**Issue by Narcha** [HULKs/hulk#457]
We sometimes lose sight of the ball (e.g. robot blocking line of sight) just when we should perform a kick-in.
Example: Today's game vs. NAO Devils, before the first goal.
Maybe we should spawn ball hypotheses where a kick-in/corner kick is likely to have occurred if we have no other hypotheses?
This needs investigation.

**Comment by schluis** [comment]
Dependent on #624

**Comment by MaikRe** [comment]
Duplicate of #514
